views/home.mako

```diff
@@ -66,7 +66,7 @@
 
 <div class="row">
     <div class="col-md-12">
-        <div class="pull-left">
+        <div id="showRootSelect" class="pull-left">
```

Comment:
@medariox what do you think about hidding the div if only ONE root folder? just need to check `backend_dirs` len